### ISSUE TITLE
Latest update seems to break a few things: extra newlines, no longer working before export

### ISSUE THREAD

**Issue by MiracleBlue** [nikhilkalige/docblockr#210]
Just updated to the latest version in my atom (also on latest stable atom 1.9.9), and now whenever I hit enter after /**, I get an extra newline.  Here's an example of exactly what happens:

```javascript
/**
 * 
 * 
 * @param {[type]} foo [description]
 * @param {[type]} bar [description]
 * @returns {[type]} [description]
 */
function doThings(foo, bar) {
    return true;
}
```

Notice the extra newline above the first param?  It also puts the cursor there, so there's this extra space above the description and I have to backspace.

Also when I hit enter to line-break a long description, it again adds an extraneous newline.

```javascript
/**
 * Some long descripion that I want to line break
 *
 * And this is where the cursor ends up after hitting enter
 * @param {[type]} foo [description]
 * @param {[type]} bar [description]
 * @returns {[type]} [description]
 */
function doThings(foo, bar) {
    return true;
}
```

Another issue that might be just my imagination is that it seems this no longer works for exported functions, and I'm fairly sure it did previously (but I could be wrong)

```javascript
/**
 *
 * 
 */
export function doThings(foo, bar) {
    return true;
}
```

![Here is a screenshot of my settings for the plugin](https://cloud.githubusercontent.com/assets/23368/17949748/44a734fc-6a9a-11e6-877c-4872a0e2be15.png)

I love this plugin and rely on it every day to help make my life easier, so many thanks to you for making it.  Let me know if you require any further information :)

**Comment by MoritzKn** [comment]
Thanks for reporting this and your nice feedback =).
The first thing is an known issue, but unfortunately we couldn't find out what causes this behavior yet.
See #202 for more information on that.

Previously exported functions where not really supported, it was kind of a bug. I improved the JS function parser which removed support for exported functions as a side effect. Anyway we already have an PR which will bring back support for exported functions and make parsing them even better: #203

**Comment by MoritzKn** [comment]
#204 is merged and release with version 0.8.4 so the second thing shouldn't be an issue anymore.

**Comment by MiracleBlue** [comment]
No worries!  Thanks for that!  I'lll follow the other issue for updates :)
